src/bootstrap/README.md

```diff
@@ -115,24 +52,6 @@ compiler. What actually happens when you invoke rustbuild is:
 The goal of each stage is to (a) leverage Cargo as much as possible and failing
 that (b) leverage Rust as much as possible!
 
-## Incremental builds
-
-You can configure rustbuild to use incremental compilation with the
-`--incremental` flag:
-
-```sh
-$ ./x.py build --incremental
-```
-
-The `--incremental` flag will store incremental compilation artifacts
-in `build/<host>/stage0-incremental`. Note that we only use incremental
-compilation for the stage0 -> stage1 compilation -- this is because
-the stage1 compiler is changing, and we don't try to cache and reuse
-incremental artifacts across different versions of the compiler.
-
-You can always drop the `--incremental` to build as normal (but you
-will still be using the local nightly as your bootstrap).
-
 ## Directory Layout
 
 This build system houses all output under the `build` directory, which looks
```

Comment:
This whole section looks partially duplicated in https://rustc-dev-guide.rust-lang.org/building/bootstrapping.html#directories-and-artifacts-generated-by-bootstrap. I think we should keep it here and remove the section in the dev guide.

cc @bjorn3 , I know you've said that section has been helpful to you in the past. Maybe we can link to here from the guide.

Reply:
I don't recall having said that this section was useful in the past. I do think it makes sense to deduplicate, but I'm not sure which copy should remain and where.